src/raylib.h

```diff
@@ -1220,6 +1220,8 @@ RLAPI void UpdateCameraPro(Camera *camera, Vector3 movement, Vector3 rotation, f
 // NOTE: It can be useful when using basic shapes and one single font,
 // defining a font char white rectangle would allow drawing everything in a single draw call
 RLAPI void SetShapesTexture(Texture2D texture, Rectangle source);       // Set texture and rectangle to be used on shapes drawing
+RLAPI Texture2D GetShapesTexture();                                     // Get texture that is used for shapes drawing
```

Comment:
C language requires specifying `void` for input parameter, no parameter is understood as any number of parameters.

Reply:
Fixed